### ISSUE TITLE
Using grape integration throws `uninitialized constant Datadog::Contrib::Rack::TraceMiddleware`

### ISSUE THREAD

**Issue by dim** [DataDog/dd-trace-rb#525]
Manually adding the following line fixes the problem:

```
require 'ddtrace/contrib/rack/middlewares'
```

**Comment by delner** [comment]
I see... probably because we switched from a literal constant to a constant referenced in the Rack tracing middleware. This line of code here:

```ruby
# set the request span resource if it's a `rack.request` span
request_span = payload[:env][Datadog::Contrib::Rack::TraceMiddleware::RACK_REQUEST_SPAN]
```

https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/contrib/grape/endpoint.rb#L74

Thanks for the report, we'll add this `require` to the Grape integration.

**Comment by delner** [comment]
This is being addressed in #533 . Should roll with 0.15.0.

**Comment by delner** [comment]
Merged to `0.15-dev`, should roll today. Thank you @dim !
